src/ember-i18n-test-helpers.js

```diff
@@ -26,7 +26,15 @@ define("ember-i18n-test-helpers", ["ember", "ember-test-helpers", "exports"], fu
 			return entry(params);
 		};
 
-		context.registry.register('service:i18n', {t: t}, DO_NOT_INSTANTIATE);
+		var exists = function existsHelper(key, params) {
+			var entry = translations[key] || defaultValue;
+			return Ember.typeOf(entry) !== 'null';
```

Comment:
Is it your intention to accept all keys as existing if a default value is set? This is hardly expected behavior from my point of view.